### ISSUE TITLE
[Bug Report]: Can't calibrate controller in system settings

### ISSUE THREAD

**Issue by I2000C** [ndeadly/MissionControl#682]
### Switch Firmware Version

16.0.2 (Latest)

### Atmosphère Version

1.5.2 (Latest)

### Mission Control Version

0.9.2 (Latest)

### Boot Method

Fusée

### Issue Description

I have 2 controllers (DS3 and DS4) which have some drift in their sticks.
However, when I try to calibrate them in system settings the screen freezes and doesn't respond to any moves of the stick, making it impossible to calibrate them.

In this screen, the controllers work:
![image](https://github.com/ndeadly/MissionControl/assets/29607352/d7534940-2c49-4304-98e0-0a2c4cf55392)
but in this screen, they doen't work:
![image](https://github.com/ndeadly/MissionControl/assets/29607352/1b67e7bc-a895-4a84-800a-3086c722e9ce)

Is this a bug or this feature isn't actually supported?

P.S.: This only happens with these 2 controllers and not with official controllers like joycons.

**Comment by ndeadly** [comment]
This definitely worked in the past. It was the motivation behind storing a virtual controller memory file for each controller under `/config/MissionControl/controllers`. I will take a look into it. Maybe something has changed in recent firmwares.

As a side note, this type of calibration might not actually fix your stick drift problem, depending on it's nature. It's meant for calibrating the centre position and range of motion, and won't fix joycon-style drift or old sticks where the centre position may drift about (you would potentially need an increased inner deadzone to deal with this)

**Comment by ndeadly** [comment]
I just checked it on my DS4. I can confirm that it appears to do nothing on that second screen. What I did notice, however, is if I move the _other_ stick then it starts registering input for the one I'm trying to calibrate after a few seconds and I'm able to complete the calibration. Absolutely no idea why this would happen though, I will need to spend some time with it.

**Comment by I2000C** [comment]
Yes, you're right, by doing that I'm able to calibrate my 2 controllers.
That's not the best solution, but it could work for the moment.
Thank you
